crates/apub/src/protocol/objects/group.rs

```diff
@@ -98,3 +99,27 @@ impl Group {
     Ok(())
   }
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub(crate) enum AttributedTo {
+  Lemmy(CollectionId<ApubCommunityModerators>),
+  Peertube(Vec<AttributedToPeertube>),
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct AttributedToPeertube {
+  #[serde(rename = "type")]
+  pub kind: PersonType,
+  pub id: ObjectId<ApubPerson>,
+}
```

Comment:
There is already `AttributedToPeertube` in page.rs with the same definition that you can use.